js/scripts.js

```diff
@@ -0,0 +1,37 @@
+var tweetLink = "https://twitter.com/intent/tweet?text=";
+var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+var prefix = "https://cors-anywhere.herokuapp.com/";
+
+function getQuote() {
+	$.getJSON(prefix + quoteUrl, createTweet);
+	$.ajaxSetup({ cache: false });
+}
+
+function createTweet(input) {
+	var data = input[0];
```

Comment:
at first check that your response is an array and it contain at least one element